packages/gatsby-remark-prismjs/README.md

```diff
@@ -392,12 +399,28 @@ unless explicitly overridden by the `promptUser` and `promptHost` options in the
 
 ### Diff code blocks
 
-You can specify language for `diff` code blocks by using `diff-[language]` to enable syntax highlighting in diffs:
+You can specify language for `diff` code blocks by using `diff-[language]` to enable syntax highlighting in diffs. Canonically, when writing code _diffs_, the code should be indented by two characters. If the code is being changed, the first character should be a `+`, `-`, or `!` otherwise left a space. The second character should always be a space. Git-style file-location coordinates need not be indented.
```

Comment:
TIL!

Reply:
You and me both! I Knew the + - but Prism implements ! — not sure if that's a Prism specific thing, and depending on your Prism theme it can be pretty subtle (not exactly the 'exclamation mark' kind of effect I'd hope for) but it _is_ there so I figured I should call it out 😀